.ado/publish.yml

```diff
@@ -69,6 +69,9 @@ jobs:
         ArmDebug:
           BuildConfiguration: Debug
           BuildPlatform: arm
+        Arm64Debug:
```

Comment:
@Khalef1 this file seems modified by your original branch.
Can your confirm these changes are intended?